javascripts/chooseplayer.js

```diff
@@ -9,16 +11,22 @@ var FoodFight = (function(choose){
     FoodFight.setUpGame(currentHero, currentEnemy);
   });
 
+  heroBtn.click(function() {
+    selected = heroBtn.filter(":checked").val();
+    console.log(selected);
+  });
+
   choose.setHero = function(){
-    var hero = FoodFight.Larry;
+    var name = userName.val()
+    var hero = new FoodFight[selected](name);
```

Comment:
correct me if I'm wrong here, but I think we may want to take the (name) off this line too, so hero is the full selected hero object!